### PR TITLE
Sign In , SignUp and Splash screen Added

### PR DESCRIPTION
**Pull request by jay-a-shah**

### CODE REVIEW

File: app/src/main/java/com/example/simformcafeteria/SignUpActivity.kt [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import kotlinx.android.synthetic.main.activity_sign_up.*
+
+class SignUpActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_sign_up)
+        if(radioButtonEmployee.isSelected){
+
+                editTextEmployeeMaterialLayout.visibility = View.VISIBLE
+
```
Comment (Purvesh-Dodiya) [inline app/src/main/java/com/example/simformcafeteria/SignUpActivity.kt]: empty lines

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -6,13 +6,113 @@
...
+        app:layout_constraintTop_toBottomOf="@id/textViewWelcome" />
+
+    <androidx.appcompat.widget.AppCompatTextView
+        android:id="@+id/textViewWelcome"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:fontFamily="@font/kavoon"
+        android:text="@string/tv_welcome_to_simform_cafeteria"
+        android:textAlignment="center"
+        android:textColor="#E64A19"
```
Comment (Purvesh-Dodiya) [inline app/src/main/res/layout/activity_main.xml]: move color to color file

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -6,13 +6,113 @@
...
+        android:textSize="30dp"
```
Comment (Purvesh-Dodiya) [inline app/src/main/res/layout/activity_main.xml]: textSize should be in sp

File: app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -6,13 +6,113 @@
...
         app:layout_constraintTop_toTopOf="parent" />
 
+    <androidx.appcompat.widget.AppCompatTextView
+        android:id="@+id/textViewSignIn"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="25dp"
+        android:layout_marginTop="10dp"
+        android:fontFamily="@font/kavoon"
+        android:text="Sign in !"
```
Comment (Purvesh-Dodiya) [inline app/src/main/res/layout/activity_main.xml]: static text

File: app/src/main/res/layout/activity_main.xml [hunk 5]
```diff
@@ -6,13 +6,113 @@
...
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_margin="25dp"
+        android:hint="@string/hint_mobile_no"
+        android:textColorHint="#E64A19"
+        app:boxCornerRadiusBottomEnd="5dp"
+        app:boxCornerRadiusBottomStart="5dp"
+        app:boxCornerRadiusTopEnd="5dp"
+        app:boxCornerRadiusTopStart="5dp"
+        app:boxStrokeColor="#434242"
```
Comment (Purvesh-Dodiya) [inline app/src/main/res/layout/activity_main.xml]: move color to color file

File: app/src/main/res/layout/activity_main.xml [hunk 6]
```diff
@@ -6,13 +6,113 @@
...
+    <androidx.appcompat.widget.AppCompatButton
+        android:id="@+id/btnSignIn"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_margin="25dp"
+        android:layout_marginTop="24dp"
+        android:background="@drawable/button_corner"
+        android:paddingStart="35dp"
+        android:paddingEnd="35dp"
+        android:text="Sign In"
```
Comment (Purvesh-Dodiya) [inline app/src/main/res/layout/activity_main.xml]: static string

File: app/src/main/res/layout/activity_sign_up.xml [hunk 7]
```diff
@@ -0,0 +1,251 @@
...
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <androidx.appcompat.widget.AppCompatTextView
+        android:id="@+id/textViewSignUp"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="25dp"
+        android:fontFamily="@font/kavoon"
+        android:text="Sign Up !"
```
Comment (Purvesh-Dodiya) [inline app/src/main/res/layout/activity_sign_up.xml]: static string

File: app/src/main/res/layout/activity_sign_up.xml [hunk 8]
```diff
@@ -0,0 +1,251 @@
...
+        android:textColorHint="@color/redSimformColor"
+        app:boxCornerRadiusBottomEnd="5dp"
+        app:boxCornerRadiusBottomStart="5dp"
+        app:boxCornerRadiusTopEnd="5dp"
+        app:boxCornerRadiusTopStart="5dp"
+        app:boxStrokeColor="@color/redSimformColor"
+        app:boxStrokeWidth="2dp"
+        app:hintAnimationEnabled="true"
+        app:hintEnabled="true"
+        app:hintTextColor="#E64A19"
```
Comment (Purvesh-Dodiya) [inline app/src/main/res/layout/activity_sign_up.xml]: same for color

File: app/src/main/res/layout/activity_sign_up.xml [hunk 9]
```diff
@@ -0,0 +1,251 @@
...
+        <com.google.android.material.textfield.TextInputEditText
+            android:id="@+id/editTextName"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:drawablePadding="10dp"
+            android:fontFamily="@font/wellfleet"
+            android:inputType="textPersonName"
+            android:textSize="20sp" />
+    </com.google.android.material.textfield.TextInputLayout>
+
```
Comment (Purvesh-Dodiya) [inline app/src/main/res/layout/activity_sign_up.xml:69]: extra empty line

File: app/src/main/res/layout/activity_sign_up.xml [hunk 10]
```diff
@@ -0,0 +1,251 @@
...
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/dropDownForEmployee" />
+
+    <androidx.appcompat.widget.AppCompatRadioButton
+        android:id="@+id/radioButtonTrainee"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="25dp"
+        android:fontFamily="@font/wellfleet"
+        android:text="Trainee"
```
Comment (Purvesh-Dodiya) [inline app/src/main/res/layout/activity_sign_up.xml]: same for text, check in all the files

**Review by Purvesh-Dodiya — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/simformcafeteria/MainActivity.kt [hunk 11]
```diff
@@ -1,7 +1,9 @@
 package com.example.simformcafeteria
 
+import android.content.Intent
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import kotlinx.android.synthetic.main.activity_main.*
```
Comment (Purvesh-Dodiya) [inline app/src/main/java/com/example/simformcafeteria/MainActivity.kt]: remove astric import

File: app/src/main/java/com/example/simformcafeteria/SignUpActivity.kt [hunk 12]
```diff
@@ -0,0 +1,17 @@
+package com.example.simformcafeteria
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.View
+import androidx.core.view.isVisible
+import kotlinx.android.synthetic.main.activity_sign_up.*
```
Comment (Purvesh-Dodiya) [inline app/src/main/java/com/example/simformcafeteria/SignUpActivity.kt]: same

**Review by Purvesh-Dodiya — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/activity_main.xml [hunk 13]
```diff
@@ -6,13 +6,112 @@
...
+        android:layout_height="wrap_content"
+        android:layout_marginStart="25dp"
+        android:layout_marginTop="10dp"
+        android:fontFamily="@font/kavoon"
+        android:text="@string/tv_sign_in"
+        android:textColor="@color/black"
+        android:textSize="35sp"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/img_simform_logo" />
+
```
Comment (purvesh-d) [inline app/src/main/res/layout/activity_main.xml]: extra empty space

File: app/src/main/res/layout/activity_main.xml (hunk 5 above)
Comment (purvesh-d) [reply app/src/main/res/layout/activity_main.xml]: pending

**Review by purvesh-d — CHANGES_REQUESTED** [review]
Also change the pattern of giving id in xml. Underscore or camel case chose any one

File: app/src/main/res/layout/activity_sign_up.xml (hunk 9 above)
Comment (purvesh-d) [inline app/src/main/res/layout/activity_sign_up.xml:70]: pending

**Review by purvesh-d — APPROVED** [review]

**Review by purvesh-d — APPROVED** [review]

**Pull request merged**